bolt/docs/CMakeLists.txt

```diff
@@ -48,14 +48,6 @@ if (LLVM_ENABLE_DOXYGEN)
     set(bolt_doxygen_qhp_cust_filter_attrs "")
   endif()
 
-  option(LLVM_DOXYGEN_SVG
```

Comment:
Will removing the option be disruptive (particularly to downstreams)? Do we need a deprecation period before we remove it?

Reply:
(Not to us in Nixpkgs at least!) Though I doubt downstream distros are the people who want to build the API docs in general.

Reply:
The couple of projects I found on Github (which sets it to "on" might get

```
CMake Warning:
  Manually-specified variables were not used by the project:
    LLVM_DOXYGEN_SVG
```

But it won't break anyone's build

Reply:
Awesome, thanks!